src/cli.rs

```diff
@@ -88,14 +88,16 @@ static VERSION_SHA: Lazy<String> = Lazy::new(|| {
     }
 });
 
+pub(crate) const DEFAULT_AMBER_YAML: &str = "amber.yaml";
+
 /// Utility to store encrypted secrets in version trackable plain text files.
 #[derive(Clap, Debug)]
 pub struct Opt {
     /// Turn on verbose output
     #[clap(short, long, global = true)]
     pub verbose: bool,
     /// amber.yaml file location
-    #[clap(long, default_value = "amber.yaml", global = true, env = "AMBER_YAML")]
+    #[clap(long, default_value = DEFAULT_AMBER_YAML, global = true, env = "AMBER_YAML")]
```

Comment:
One downside to this approach: we cannot distinguish "explicit stated `amber.yaml`" from "didn't state anything." It's not a big deal, but I think an approach that captures that concept would be better. I took a stab at adding a method `find_amber_yaml` to `Opt` instead, which I think also addresses my concern below about the `save` functionality not being updated:

5847b853475fa35c899a7cf0080d6ee20a9d761b

I'm not thrilled about the need for `&mut self`, but it's a reasonable thing to require in this case. What do you think?

Reply:
Thanks, the changes looks good to me. And yes, modelling it explicitly via Option is much better. I like your changes - I have left a single comment there.

Reply:
Cool, want to pull those changes into your branch to update this PR?

Reply:
Sure, Updated. Thanks!